dataprofiler/profilers/utils.py

```diff
@@ -759,3 +759,20 @@ def merge_profile_list(list_of_profiles, pool_count=5):
 
     list_of_profiles[0]._restore_data_labelers(data_labeler)
     return list_of_profiles[0]
+
+
+def combine_means(mean1, mean2, n1, n2):
+    """Combine two mean vectors
+
+    :param mean1: First mean vector
+    :type mean1: np.ndarray
+    :param mean2: Second mean vector
+    :type mean2: np.ndarray
+    :param n1: Number of samples mean1 was calculated from
+    :type n1: int
+    :param n2: Number of samples mean2 was calculated from
+    :type n2: int
+    :return: Combined mean vector
+    :rtype: np.ndarray
+    """
+    return (mean1 * n1 + mean2 * n2) / (n1 + n2)
```

Comment:
why not `return (mean1 + mean2) / 2`?

Reply:
That equation would only work if `n1 == n2`. If sample sizes the means were calculated from are different then we need to use `(mean1 * n1 + mean2 * n2) / (n1 + n2)`

Reply:
Ref: https://www.statisticshowto.com/combined-mean/

Reply:
not weighted mean